### ISSUE TITLE
Add error message to delete data folder if LFS detect already data in the DAG

### ISSUE THREAD

**Issue by tgrospic** [rchain/rchain#3746]
## Overview

When syncing LFS we should detect if DAG storage have already stored blocks in which case we want throw exception and stop the node.

**Comment by hilltracer** [comment]
@tgrospic can I add this checking in this place?
https://github.com/rchain/rchain/blob/b89d6ec56ce437727aafcdf3c4dc21c3262f36ec/casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala#L115-L125

Something like:
```scala
if (dag.dagSet.nonEmpty && !standalone)
          new RuntimeException("DAG storage has already stored blocks. LFS syncing is not possible. Please remove DAG data from Rnode folder.").raiseError
```

**Comment by tgrospic** [comment]
The right place is when node is starting with syncing mode. Standalone or not error should be thrown is DAG is not empty.
https://github.com/rchain/rchain/blob/b89d6ec56ce437727aafcdf3c4dc21c3262f36ec/casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala#L80-L96

I've changed the message a little bit.
`DAG storage has already stored blocks. LFS syncing is not possible. Please remove old data files from RNode folder.`

**Comment by hilltracer** [comment]
@tgrospic I added this validation in `startSyncingMode()` but it doesn't work because it doesn't process here:
https://github.com/rchain/rchain/blob/b89d6ec56ce437727aafcdf3c4dc21c3262f36ec/casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala#L115-L125
if we have `dag.dagSet.isEmpty && !standalone` -> startSyncingMode() doesn't call.
So, I need change top code (as I suggest before). We have 4 different situations:
1) dag.dagSet.isEmpty && standalone    => createStoreBroadcastGenesis
2) !dag.dagSet.isEmpty && standalone   => ???
3) dag.dagSet.isEmpty && !standalone   => startSyncingMode
4) !dag.dagSet.isEmpty && !standalone  => Error
Please clarify what I should to do in case 2?

**Comment by tgrospic** [comment]
You are right, inside `startSyncingMode` is too late for this check. Let's keep this issue on hold until new LFS bootstrap code will be merged.
